modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb

```diff
@@ -9,9 +9,10 @@ module Forms
       class DisabilityCompensationController < BaseFormController
         FORM_NUMBER = '526'
         before_action :verification_itf_expiration, only: [:submit_form_526]
+        before_action :validate_526_payload, only: [:submit_form_526]
         skip_before_action(:authenticate)
         skip_before_action(:verify_power_of_attorney)
-        skip_before_action :validate_json_schema, only: [:upload_supporting_documents]
+        skip_before_action :validate_json_schema, only: %i[upload_supporting_documents]
```

Comment:
some inconsistency, this one uses `%i[` the other just uses `[`

Reply:
This is because our rubocop rules require %i if there is more than one item, so I updated the single item arrays to also use %i

Reply:
yeah that sounds good to me!